eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -70,9 +70,9 @@ jobs:
     # Linux arm64
     - ${{ if eq(parameters.platform, 'linux_arm64') }}:
       - ${{ if eq(variables['System.TeamProject'], 'public') }}:
-        - (Ubuntu.2404.Arm64.Open)Ubuntu.2204.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-24.04-helix-arm64v8
+        - (Ubuntu.2404.Arm64.Open)Ubuntu.2204.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-helix-arm64v8
```

Comment:
Should the name in parens at the start also be changed to 2204 instead of 2404?

Reply:
I think it should generally be ok to have mismatched host/docker image here. Let's see what linux-arm64 testing says.

Reply:
This one is actually an optional queue alias, so will result in some confusing logs (but shouldn't cause any harm) I don't really want to restart the CI runs to fix it... Hopefully should be ok given this is temporary.